design/open/delta-catalog-exporter.md

```diff
@@ -0,0 +1,83 @@
+# Delta Lake catalog exporter
+
+## Introduction
+
+The Delta Lake table format manages its catalog of used files through a log-based system. This log, as the name implies,
+contains a sequence of deltas representing changes applied to the table. The list of files that collectively represent
+the table at a specific log entry is constructed by reapplying the changes stored in the log files one by one, starting
+from the last checkpoint (a file that summarizes all changes up to that point) and progressing to the latest log entry.
+Each log entry contains either an `add` or `remove` action, which adds or removes a data (parquet) file, ultimately
+shaping the structure of the table.
+
+In order to make Delta Lake tables accessible to external users, we aim to export the Delta Lake log to an external
+location, enabling these users to read tables backed by lakeFS and Delta Lake.
+
+---
+
+## Proposed Solution
+
+Following the [catalog exports issue](https://github.com/treeverse/lakeFS/issues/6461), the Delta Lake log will be
+exported to the `${storageNamespace}/_lakefs/exported/${ref}/${commitId}/${tableName}/_delta_log/` path, which resides 
+within the user's designated storage bucket.  
+Within the `_delta_log` directory, you will find the following components:
+1. The last checkpoint (or the initial log entry if no checkpoint has been established yet).
+2. All the log entries that have been recorded since that last checkpoint.
```

Comment:
Observation: With this structure, the number of exported files is not tightly coupled with the number of changes to the table. Assuming N is the number of log entries between checkpoints, adding a single file to the table can result in `N + 1 + 1` objects to export. Assuming `N=10`, we should be okay (but worth mentioning in the docs)